lib/services/access_request.go

```diff
@@ -1328,7 +1350,7 @@ func MarshalAccessRequest(accessRequest types.AccessRequest, opts ...MarshalOpti
 //     should be satisfied by exactly 1 role. The first such role will be
 //     requested, all others will be pruned unless they are necessary to access
 //     a different resource in the set.
-func (m *RequestValidator) pruneResourceRequestRoles(
+func (m *RequestValidator) PruneResourceRequestRoles(
```

Comment:
I don't see why this needs to be exported

Reply:
leftover from past implementation, revised.